src/resources/Constants.java

```diff
@@ -0,0 +1,9 @@
+//user defined package
+package resources;
+
+public class Constants {
```

Comment:
This class is just a constant class and again doesnt need a public constructor.Make appropriate changes